.github/workflows/python-publish.yml

```diff
@@ -4,27 +4,28 @@
 name: Upload Python Package
 
 on:
-    release:
```

Comment:
See if you can figure out a way to make this change without reformatting the whole file. If you have to reformat it, do that on a separate commit.

Reply:
I believe I fixed it. it was pre-commit.